src/type/__tests__/scalars-test.ts

```diff
@@ -280,6 +283,7 @@ describe('Type System: Specified scalar types', () => {
       expect(coerceOutputValue('-1.1')).to.equal(-1.1);
       expect(coerceOutputValue(false)).to.equal(0.0);
       expect(coerceOutputValue(true)).to.equal(1.0);
+      expect(coerceOutputValue(1n)).to.equal(1n);
```

Comment:
```suggestion
      expect(coerceOutputValue(1n)).to.equal(1);
```

Assuming we take the change below.